transcripts/share-apis/code-browse/search.json

```diff
@@ -4,7 +4,7 @@
       "avatarUrl": null,
       "handle": "test",
       "name": null,
-      "tag": "User",
+      "tag": "user",
```

Comment:
Both `org` and `user` are now lowercase, which is fine by me, but can we also make the `Project` variant lowercase?

Reply:
Done!